src/report_reader.cpp

```diff
@@ -349,22 +349,22 @@ DataFrame<T> ReportReader<T>::Population::get(const nonstd::optional<Selection>&
         node_ids = selection->flatten();
     }
 
+    std::vector<std::pair<uint64_t, uint64_t>> positions;
```

Comment:
This generally helps with readability, although in this particular case it can be confusing. Do you have a name in mind for the alias?

For instance, if we change it here, we must change it in the `std::map<<NodeID, std::pair<uint64_t, uint64_t>> nodes_pointers_`.

Reply:
Yeah, I meant for it to be everywhere.  It can be internal to the `SpikeReader`, and something similar to the alias in population.h: `using Range = std::pair<uint64_t, uint64_t>`.  Was just a thought, up to you.